build/ci/community/wait_demo_pr_checks.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash -xe
+set +o pipefail
+
+gh_pr_checks_command (){
+  community_operators_branch=$1
+  forked_repository=$2
+  gh pr checks $community_operators_branch --repo $forked_repository
+}
+
+wait_for_checks_to_start(){
+  community_operators_branch=$1
+  forked_repository=$2
+  while [ `gh_pr_checks_command $community_operators_branch $forked_repository | wc -l` -eq 0 ]; do
+    sleep 1
+  done
+}
+wait_for_checks_to_complete(){
```

Comment:
do we want to have a single line break between function? what is the convention here?
please try to use an IDE that helps with such conventions